.github/workflows/cd_ios.yml

```diff
@@ -43,6 +43,48 @@ jobs:
           cd examples/mobile/ios && bundle install
           bundle exec fastlane build_and_deploy_to_firebase --env enterprise build_number:$GITHUB_RUN_NUMBER
 
+  Build-And-Deploy-To-TestFlight-InternalTest:
+    runs-on: macos-12
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+
+      - name: Install modules
+        run: yarn install
+
+      - name: Setup bundler
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
+
+      - name: Assign internal environment
+        env:
+          INTERNAL_RELEASE_TOKEN_URLS: ${{secrets.INTERNAL_RELEASE_TOKEN_URLS}}
+        run: |
+          echo $INTERNAL_RELEASE_TOKEN_URLS > packages/mobile/src/providers/TokenUrl.tsx
```

Comment:
Is it possible to read this secret directly from project env variables instead from file by further steps of build process?

Reply:
Hi @msobo1 - This secret is written into this source file and it drives a run time logic in the app on where(URL) to fetch the access token for the app from. This gets built into the App binary. So writing to the source file is needed.

Does that answer your question ?

Reply:
Ok, we understand. What actions can be taken with "INTERNAL_RELEASE_TOKEN_URLS" token ? What kind of access/privileges does it give to potential attacker ?

Reply:
@artur-l - That's a good question. The URLs issue a test access token to the potential attacker, with which they can initialise and use the Dolby.io SDK's. They don't even need to sign up as a user into Dolby.io. 

I am not sure at this point if there are any usage restrictions / difference in privilege for the `token's issued by the test server`. 
@CasperMok / @RameshDPrasad - Do you know ?

Reply:
Hi @artur-l - Loosing the test server URLs to a hacker means they can use the Dolby.io services using our SDK's for free without paying.

Reply:
Everything used/created for testing/debugging during development phase should stay there - we can't allow potential attacker to use Dolby.io for free. We need to remove the token from public/production build, so any bug hunter won't see the string "TOKEN" in a public binary. The best option would be to remove it from production build process (so creating development pipeline is also needed if you need specific testing phase).

Reply:
Hi Artur - The intend for this build (Distributed via TestFlight/Playstore) is for internal testing only - ie, only to a limited group of Dolby employees. There is no way by which a public customer can get hold of this build.

Reply:
Ok, that's clear. We accept that solution. Please put a short comment about INTERNAL_RELEASE_TOKEN_URLS variable usage so it's clear when checking the yaml code.

Reply:
Done, please check.